internal/pkg/service/buffer/sink/tablesink/storage/volume/assignment/assigner_test.go

```diff
@@ -37,31 +36,31 @@ func TestVolumes_VolumesFor(t *testing.T) {
 		{
 			Name:            "empty",
 			Count:           1,
-			PreferredTypes:  []string{},
+			PreferredTypes:  []string{"missing"},
```

Comment:
Added validation, there must be at least one configured preferred type - updated tests.